sim/warlock/destruction/TestDestruction.results

```diff
@@ -1721,8 +1742,8 @@ dps_results: {
 dps_results: {
  key: "TestDestruction-AllItems-VestmentsoftheFacelessShroud"
  value: {
-  dps: 35497.41631
-  tps: 20804.38168
+  dps: 35168.19432
+  tps: 20636.46217
  }
```

Comment:
This one needs a look as well for both Affliction and Destruction.

Reply:
`WarlockSpellShadowBurn` vs `WarlockSpellSoulBurn`, oops.